lib/cocoapods-packager/pod_utils.rb

```diff
@@ -38,6 +51,37 @@ def install_pod(platform_name, sandbox)
         static_installer
       end
 
+      def vendored_libraries(installer, root)
+        libs = []
+        installer.pod_targets.each do |target|
+          next if target.product_module_name == @spec.name
+
+          target.file_accessors.each do |file_accessor|
+            file_accessor.vendored_static_frameworks.each do |framework|
+              name = File.basename(framework, '.framework')
+              framework_path = framework.relative_path_from(target.sandbox.root)
+              libs << File.join(root, framework_path, name)
+            end
+
+            file_accessor.vendored_static_libraries.each do |library|
+              library_path = library.relative_path_from(target.sandbox.root)
+              libs << File.join(root, library_path)
+            end
+          end
+        end
+        libs
+      end
+
+      def static_libraries(installer, root)
+        installer.pod_targets.map do |target|
+          next if target.product_module_name == @spec.name
+
+          if target.should_build?
+            File.join(root, "lib#{target.product_module_name}.a")
+          end
+        end
```

Comment:
To remove redundant dependencies I assume?  In case multiple deps have the same binary deps?